en/08_Changelogs/6.1.0.md

```diff
@@ -174,6 +174,7 @@ This functionality will not work in your front-end forms out of the box as the r
 - The [`File`](api:SilverStripe\Assets\File) class now has a `IsFolder` generated column which is included in a new index. This makes sorting files in the asset admin faster.
 - Previously there were a large number of unnecessary AJAX requests made to fetch the form schema for the search form for sections of the CMS that are searchable such as the site tree i.e. the list of pages on `/admin/page`. This has been fixed so these requests are only made when the filter button is clicked. Note this enhancement was originally released as a patch for CMS 5.4.
 - There have been a number of other smaller performance enhancements that have been included in this release. Some of these enhacements were also released as patch releases for CMS 5.4.
+- Inside your cache directory, sub-directories now include the PHP version, e.g. my-user-8.3.23/. Previously the PHP version was not included. If you do not have a silverstripe-cache/ directory in your project root, the parent cache directory that gets created no longer contains the PHP version (e.g. it will be /tmp/silverstripe-cache-var-www-html/ where it used to be /tmp/silverstripe-cache-php8.3.23-var-www-html/).
```

Comment:
```suggestion
- Inside your cache directory, sub-directories now include the PHP version, e.g. `my-user-8.3.23/`. Previously the PHP version was not included. If you do not have a `silverstripe-cache/` directory in your project root, the parent cache directory that gets created no longer contains the PHP version (e.g. it will be `/tmp/silverstripe-cache-var-www-html/` where it used to be `/tmp/silverstripe-cache-php8.3.23-var-www-html/`).
```
Just adding some backticks around folder name examples.